sdk/storage/azure-storage-file-share/pom.xml

```diff
@@ -85,6 +90,18 @@
       <version>2.0-M4-groovy-2.5</version> <!-- {x-version-update;org.spockframework:spock-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-blob</artifactId>
+      <version>12.12.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
```

Comment:
I feel like we should depend on latest of blob just cause that is usually what we do within the storage package. What do you think?

Reply:
I don't think it makes a difference for this use case. We certainly can if we want to keep things pretty and updated.